docs/docs/reference-react-component.md

```diff
@@ -88,7 +163,13 @@ The `displayName` string is used in debugging messages. JSX sets this value auto
 
```

Comment:
> If `props.color` is not provided, it will be set by default to `'blue'`.

Maybe mention that defaultProps are only used for undefined props, but not for null props.